### PR TITLE
feat: drop support of php < 7, modernise code and tests, replace travis with github actions

### PR DESCRIPTION
**Pull request by Chris53897**
Testruns: https://github.com/Chris53897/zip-code-validator/actions/runs/4096520227

### CODE REVIEW

File: src/ZipCodeValidator/Constraints/ZipCode.php [hunk 1]
```diff
@@ -2,48 +2,27 @@
...
 /**
- * Class ZipCode
  * @Annotation
  * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ * TODO NamedArgumentConstructor
  * @author Martin Schindler
- * @package ZipCodeValidator\Constraints
  */
+##[Attribute(Attribute::TARGET_PROPERTY | Attribute::TARGET_METHOD)]
+# TODO Needs rewrite of __construct() and tests
```
Comment (barbieswimcrew) [inline src/ZipCodeValidator/Constraints/ZipCode.php]: What about TODO here?

File: src/ZipCodeValidator/Constraints/ZipCode.php [hunk 2]
```diff
@@ -2,48 +2,27 @@
...
 
+use Attribute;
 use Symfony\Component\Validator\Constraint;
 use Symfony\Component\Validator\Exception\MissingOptionsException;
 
 /**
- * Class ZipCode
  * @Annotation
  * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ * TODO NamedArgumentConstructor
```
Comment (barbieswimcrew) [inline src/ZipCodeValidator/Constraints/ZipCode.php]: What about TODO here?

**Comment by barbieswimcrew** [comment]
Hi @Chris53897 , thanks for your participation. Looks good so far. Please have a look at the TODO comments you placed and let me know.

**Comment by Chris53897** [comment]
@barbieswimcrew 
Thanks for having a look.
I reverted the TODO changes, and made a issue.
https://github.com/barbieswimcrew/zip-code-validator/issues/48

**Pull request merged**